init-plugin.sh

```diff
@@ -39,14 +37,18 @@ echo "Prefix: $prefix"
 echo "NS: $namespace"
 echo "Class: $class"
 
-git clone --recursive https://github.com/xwp/wp-foo-bar.git "$slug"
+git clone "$(dirname "$0")" "$slug"
```

Comment:
This change will be helpful as you can create a new plugin without an internet connection. So you can do:

```
cd wp-content/plugins
./foo-bar/init-plugin.sh "My Awesome Plugin"
```